lib/js_browser.mli

```diff
@@ -17,6 +17,61 @@ module Storage : sig
   val clear: t -> unit
 end
 
+module Date : sig
+  type t = private Ojs.t
+  val t_of_js: Ojs.t -> t
+  val t_to_js: t -> Ojs.t
+
+  val new_date: int -> t [@@js.new]
+  val now : unit -> int [@@js.global "Date.now"]
```

Comment:
Indeed, the [manual](https://ocsigen.org/js_of_ocaml/manual/) says:
> integers are 32 bits (rather than 31 bits or 63 bits), wich [sic] is their natural size in JavaScript

Indeed, I've seen `int_of_string` fail with a date initially generated by `now`. I switched the representation to `float`, and things seem to work much better.